packages/framework/src/Model/Localization/Localization.php

```diff
@@ -90,4 +97,16 @@ public function getAdminEnabledLocales(): array
 
         return $enabledLocales;
     }
+
+    /**
+     * @param string $locale
+     */
+    public function checkLocaleIsSupported(string $locale): void
+    {
+        $allLocales = $this->getLocalesOfAllDomains();
```

Comment:
As discussed on Slack, we could be able to limit the available locales for the administration. 
I suggest unifying it into a single parameter shopsys.allowed_admin_locales (or something like that). And the first should also be the default admin locale, what do you think?